pyproject.toml

```diff
@@ -6,7 +6,8 @@ authors = [
     {name = "Evgeny Ivanov", email = "ivanov.evgeny.n@gmail.com"},
 ]
 dependencies = [
-    "ibis-framework[duckdb]>=7,<10",
+    "ibis-framework>=7,<10",
+    "narwhals>=1,<2",
```

Comment:
nice, thanks for using Narwhals!

May I suggest not setting an upper-bound?

- in part because it's generally not considered a best practice for libraries https://iscinumpy.dev/post/bound-version-constraints/
- in part because if you use `narwhals.stable.v1`, then it shouldn't be necessary https://narwhals-dev.github.io/narwhals/backcompat/

Reply:
Thank you for the suggestion. Yes, probably it's a good idea to remove upper bounds. I will do it in the next version.

Reply:
Done in #111